cmd/zoekt-sourcegraph-indexserver/default_grpc_service_configuration.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "./json_schemas/ServiceConfig.json",
+  "method_config": [
+    {
+      "name": [
+        {
+          "service": "frontend.indexedsearch.v1.IndexedSearchConfigurationService"
+        }
+      ],
+
+      "retry_policy": {
+        "max_attempts": 4,
+        "initial_backoff": "1s",
+        "max_backoff": "30s",
+        "backoff_multiplier": 2,
+        "retryable_status_codes": [
+          "UNAVAILABLE",
```

Comment:
@camdencheek I did my best to guess the status codes that would be appropriate to blindly retry on. See https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Reply:
The status codes have been tweaked as requested